devtools/generate_batch_config.py

```diff
@@ -51,14 +53,11 @@ def to_config(
                             "environment": {"variables": env_dict},
                         }
                     ],
-                    "computeResource": {"cpuMilli": 8000, "memoryMib": 32768},
-                    "volumes": [
-                        {
-                            "deviceName": "local-ssd",
-                            "mountPath": "/home/mambauser/pudl_work/output",
-                            "mountOptions": "rw,async",
-                        }
-                    ],
+                    "computeResource": {
+                        "cpuMilli": 8000,
+                        "memoryMib": 32768,
+                        "bootDiskMib": 204800,
```

Comment:
holycrap how do we end up needing 200GB?

Reply:
This was just a guess since we ran out of a small boot disk of 10GB. I'll shrink it down to 80GB to match the existing VM.

Reply:
I was just surprised. Disk is cheap! We do have a lot of data.